lib/fathead/nodejs_ref/parse.js

```diff
@@ -28,7 +28,7 @@ function getAbstract(obj) {
         stability = '<p>Stability: ' + obj.stability + ' - ' + obj.stabilityText + '</p>';
     }
 
-    a = code + a + stability;
+    a = '<section class="prog__container">' a + code + stability + '</section>';
```

Comment:
thnx moollaza
nodejs api reference file all.json as been updated ('methods' key no longer present)
need to update the code to reflect the changes